### PR TITLE
docs: update tools readme #2886

### PR DESCRIPTION
**Pull request by raphi011**
## Basics

These points need to be fulfilled for every PR:

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy.
- [ ] The PR is rebased with current master.

If you have any troubles fulfilling these criteria, please write
about the trouble as comment in the PR. We will help you.
But we cannot accept PRs that do not fulfill the basics.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned every code not directly written by me in [THIRD-PARTY-LICENSES](doc/THIRD-PARTY-LICENSES)

## Review

Reviewers will usually check the following:

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

If you are already Elektra developer:

- Add the "work in progress" label if you do not want the PR to be reviewed yet.
- Add the "ready to merge" label **if the basics are fulfilled** and you also
  say that everything is ready to be merged.

https://github.com/ElektraInitiative/libelektra/issues/2886

### CODE REVIEW

File: src/tools/README.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ The graphical user interface (GUI) is called `kdb qt-gui`.
 
 ## Web-UI
 
-For information about the Web-UI, please read [here](web).
+For information about the Web-UI, please read [here](../../doc/tutorials/install-webui.md).
```
Comment (markus2330) [inline src/tools/README.md:18]: Better link both?

**Comment by mpranj** [comment]
Thank you for improving the docs!

If this is ready, can you please rebase such that jenkins runs through?

**Comment by raphi011** [comment]
since when does a build fail when code coverage doesn't improve? this doesn't really make sense for documentation PR's

**Comment by markus2330** [comment]
It is not about not improving but if it drops a lot. But this case looks to me like an issue with coveralls. @sanssecours do you have an idea?

**Comment by mpranj** [comment]
You did not add release notes so the while build failed. Thus code coverage is at 0%.

**Comment by raphi011** [comment]
Build is green now, can be merged

**Comment by mpranj** [comment]
Thank you!

**Pull request merged**